_index.md

```diff
@@ -0,0 +1,104 @@
+
+The sessions of the SSS **TODO-track-name** track are:
+
+- [Session 01: Name](session-name-1/index.md)
+- TODO
+- TODO
+
+### Session Folder Structure
+
+A session folder consists of:
+
+- `index.md`: session contents, written in [GitHub Markdown](https://guides.github.com/features/mastering-markdown/)
+- `activities/`: each activity has its own folder with support files (including the reference solution)
+- `assets/`: support files for session contents (images, diagrams, code snippets, demos, packet captures, etc.)
+- `slides/`: Markdown file used to generate the session slides.
+
+### Activity Folder Structure
+
+An activity folder consists of:
+
+- `README.md`: details the challenge name, challenge description, points, hints, vulnerability overview, solution overview.
+- `flag` (optional): contains the flag string.
+- `public/`: stores public files that are to be packed in an archive that is to be made available to participants.
+  They are usually generated from the `src/` folder, though they may also be static files.
+- `sol/`: the reference solution of the activity.
+- `src/` (optional): stores similar contents to a CTF challenge.
+  The generated files that are to be made available to participants will be stored in the `public/` folder in the corresponding activity folder in the public repository.
+  Most activities require a source code folder, but some may not, making the `src/` folder optional.
+- `deploy/` (optional): stores similar contents to a CTF challenge.
+  Most activities require remote deployment in a Docker container, but some may not, making the `deploy` folder optional.
+
+  It typically consists of the following files:
+
+  - `Dockerfile`: for creating the Docker container that will run the challenge on the remote system.
+  - `docker-compose.yml`: for configuring the Docker containers
+  - `Makefile`: the building and running of Docker containers.
+    This file should include `common/activity.mk` in most cases, but if a challenge has a different structure it can implement its own Makefile.
+    For the general Makefile we have the following commands:
+    - `make run`: start the Docker container
+    - `make build`: build the Docker image
+    - `make generate`: generate the file with the flag from `../flag`
+    - `make stop`: stop the container
+    - `make clean`: remove the image, the container and the generated file
+  - `run.sh` (when required): Docker startup script, used to start services.
+    It's used inside the `Dockerfile` with `COPY + CMD` commands.
+  - Additional files required for the deployment.
+    They may be static files or files generated from the `src/` folder.
+
+The `session-name-1/activities/activity-folder-1/` folder contains scaffolding contents of an activity folder:
+
+```
+|-- deploy/
+|   |-- .dockerignore
+|   |-- docker-compose.yml
+|   |-- Dockerfile
+|   |-- Makefile
+|   `-- run.sh
+|-- flag
+|-- README.md
+|-- sol/
+|   `-- README.md
+`-- src/
+    `-- index.template.php
```

Comment:
This doesn't belong here. It's used for web only.

Reply:
I don't think it's a problem. It's a good template for most of the activities.